### PR TITLE
Mbus alt

### PR DESCRIPTION
**Pull request by dupondje**
Alternative implementation of #141

Instead of having a obis reference for each MBUS, only have 1 generic one.

This because at least the BE and the NL DSMR devices use MBUS.
Therefor we make the MBUS obis reference generic, and convert the V5
telegram to use MBUS values.

Also remove BELGIUM_MAX_POWER_PER_PHASE as this was a duplicate.

Finally set self.maxDiff = None to have a full test output on some
tests.

This new code can then be used in Home-Assistant to add MBUS devices.

### CODE REVIEW

**Comment by jbouwh** [comment]
@dupondje  May be you can improve PR description a bit and explain why we are proposing these changes.

File: dsmr_parser/obis_references.py [hunk 1]
```diff
@@ -43,6 +43,9 @@
 INSTANTANEOUS_CURRENT_L1 = r'^\d-\d:31\.7\.0.+?\r\n'
 INSTANTANEOUS_CURRENT_L2 = r'^\d-\d:51\.7\.0.+?\r\n'
 INSTANTANEOUS_CURRENT_L3 = r'^\d-\d:71\.7\.0.+?\r\n'
+FUSE_THRESHOLD_L1 = r'^\d-\d:31\.4\.0.+?\r\n'  # Applicable when current limitation is active
+FUSE_THRESHOLD_L2 = r'^\d-\d:51\.4\.0.+?\r\n'  # Applicable when current limitation is active
+FUSE_THRESHOLD_L3 = r'^\d-\d:71\.4\.0.+?\r\n'  # Applicable when current limitation is active
```
Comment (jbouwh) [inline dsmr_parser/obis_references.py:48]: Maybe these should be added in a separate PR?

File: dsmr_parser/obis_references.py [hunk 2]
```diff
@@ -88,38 +108,6 @@
 BELGIUM_CURRENT_AVERAGE_DEMAND = r'^\d-\d:1\.4\.0.+?\r\n'
 BELGIUM_MAXIMUM_DEMAND_MONTH = r'^\d-\d:1\.6\.0.+?\r\n'
 BELGIUM_MAXIMUM_DEMAND_13_MONTHS = r'^\d-\d:98\.1\.0.+?\r\n'
-BELGIUM_MAX_POWER_PER_PHASE = r'^\d-\d:17\.0\.0.+?\r\n'  # Applicable when power limitation is active
-BELGIUM_MAX_CURRENT_PER_PHASE = r'^\d-\d:31\.4\.0.+?\r\n'  # Applicable when current limitation is active
```
Comment (jbouwh) [inline dsmr_parser/obis_references.py:92]: This seems not related to M-BUS, can we move this to another PR?

File: dsmr_parser/obis_references.py [hunk 3]
```diff
@@ -142,5 +130,3 @@
 EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q2 = r'^\d-\d:6\.7\.0.+?\r\n'
 EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q3 = r'^\d-\d:7\.7\.0.+?\r\n'
 EON_HU_INSTANTANEOUS_REACTIVE_POWER_Q4 = r'^\d-\d:8\.7\.0.+?\r\n'
-EON_HU_MAX_POWER_ON_L2 = r'^\d-\d:51\.4\.0.+?\r\n'
-EON_HU_MAX_POWER_ON_L3 = r'^\d-\d:71\.4\.0.+?\r\n'
```
Comment (jbouwh) [inline dsmr_parser/obis_references.py:146]: Same here, not sure why this is removed. May be move this to another PR?

File: dsmr_parser/telegram_specifications.py [hunk 4]
```diff
@@ -1233,7 +1132,7 @@
             # This seems to be wrong in documentation, it's not 0-0:96.50.68, but 0-0:96.3.10
         },
         {
-            'obis_reference': obis.BELGIUM_MAX_POWER_PER_PHASE,
```
Comment (jbouwh) [inline dsmr_parser/telegram_specifications.py:1236]: Seem unrelated to MBUS, can this be moved to another PR?

**Review by jbouwh — CHANGES_REQUESTED** [review]
Lest move all non MB related changes to another PR.
Looks good overall

File: dsmr_parser/obis_references.py (hunk 1 above)
Comment (dupondje) [reply dsmr_parser/obis_references.py:48]: It's because this PR just contains 2 commits. One is the mbus change and the other is #139 . Just for the ease to not need to rebase after merge of the first PR.

File: dsmr_parser/obis_references.py (hunk 3 above)
Comment (dupondje) [reply dsmr_parser/obis_references.py:146]: See above :)

File: dsmr_parser/telegram_specifications.py (hunk 4 above)
Comment (dupondje) [reply dsmr_parser/telegram_specifications.py:1236]: See above :)

File: dsmr_parser/obis_references.py (hunk 1 above)
Comment (jbouwh) [reply dsmr_parser/obis_references.py:48]: I do not agree this is right. It is just not clean as you do not need that commit here. It that were the case then this PR should be a draft.

**Review by jbouwh — APPROVED** [review]
Nice cleanup!
Thnx @dupondje 👍

**Comment by dupondje** [comment]
@ndokter : Don't know if you want to review it also? Otherwise I'll merge it in the coming days.

**Comment by dupondje** [comment]
@ndokter : this one is also merged in 1.4.0, you add it to the changelog? :)

**Comment by ndokter** [comment]
@dupondje oops, added it!
